.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
Tak z czystej ciekawości - od narodzin projektu używam v10.16. Umawiamy się teraz na wersję 12 i się tego trzymamy czy może dodajemy też 10.x tutaj? :)

Reply:
Jeśli Ci nie przeszkadza to możemy się umówić na wersję 12. Oczywiście jak chcesz to możemy zostać przy 10.x :)

Reply:
Jeśli chodzi o mnie, to 12 jest w porządku. :)
Sprawdziłem jednak, że pipeline'y na Azure dla front-endu wciąż działają i się buildują po mergach: [link](https://dev.azure.com/lightnig-team-2/Ekostraz/_build/results?buildId=74&view=logs&jobId=275f1d19-1bd8-5591-b06b-07d489ea915a&taskId=96b37e98-490b-5755-db43-568ff4a0ab3b&lineStart=17&lineEnd=17&colStart=15&colEnd=49)

Jest tam napisane `npm/6.10.1 node/v10.16.0 win32 x64`, więc myślę, że przydałoby się mieć też wersję z tych pipeline'ów podłączoną, ale z tego co widziałem, to można dodać do
```
matrix:
   node-versions: []
```
parę wersji więc to chyba nie problem? Co o tym myślisz? :)

Reply:
Myślę, że nie ma sensu wspierać tak starej wersji jak 6. Coś się zepsuje i po co mamy naprawiać to dla Node 6. Zmieńmy wersję w pipeline'ach na nowszą. To że ktoś w czasie hackathonu ustawił tam wersję 6 to nie znaczy, że musimy się tego trzymać.
Dodałem wersję 10 do builda

Reply:
Chyba drobne nieporozumienie - w poprzednim poście wkleiłem z pipeline'ów kawałek i tam Node jest w wersji 10.16 a npm v6, tak więc o Node 6 nic nie pisałem i zgadzam się z Tobą, że nie ma co go wspierać :)

Pipeline'y są na Azure i nie wiem czy ktoś się nimi zajmuje obecnie, więc kiedyś trzeba będzie pewnie i tak do nich wrócić i wtedy można na 12 zmienić :)

Chcesz już mergować brancha czy czekamy z tym aż kiedyś testy naprawimy? :) Ja jestem za mergowaniem, ewentualnie się odkomentuje tą linijkę z testami jak testy naprawimy.

Reply:
A, rzeczywiście, to był NPM 6. Sorry :)
Tak, zmergujmy teraz, a testy włączymy jak będą działać.